src/Utilities/ToolTask.cs

```diff
@@ -699,6 +699,9 @@ string commandLineCommands
                 // Start the process
                 proc.Start();
 
+                // Execute process
+                this.ProcessStarted();
```

Comment:
Should this be after the `StandardInput.Dispose()` invocation below? My instinct is to say "yes", because the overrider might do something similar and we'd rather it throw in their code than ours.

Reply:
@olgaark do you have an opinion on this?

Reply:
I think either should be fine for our specific purposes.